ee/codegen/src/__test__/nodes/__snapshots__/inline-prompt-node.test.ts.snap

```diff
@@ -285,222 +285,6 @@ class PromptNode(InlinePromptNode):
 "
 `;
 
-exports[`InlinePromptNode > FUNCTION_DEFINITION block type > basic > getNodeDisplayFile for FUNCTION_DEFINITION block type 1`] = `
```

Comment:
Removes some snapshots whose tests were actually removed in a prior PR but `npm run test:update` hasn't been run since.